README.md

```diff
@@ -187,21 +191,28 @@ The Living Documentation Generator action provides a key output that allows user
 The Living Documentation Generator is designed to produce an Issue Summary page (index.md) along with multiple detailed single issue pages.
 
 ### Index page example
+
 ```markdown
 # Issue Summary page
 
 Our project is designed with a myriad of issues to ensure seamless user experience, top-tier functionality, and efficient operations.
 Here, you'll find a summarized list of all these issues, their brief descriptions, and links to their detailed documentation.
 
 ## Issue Overview
-| Organization name| Repository name            | Issue 'Number - Title'                        | Linked to project | Project Status | Issue URL  |
-|------------------|----------------------------|-----------------------------------------------|-------------------|----------------|------------|
-|AbsaOSS           | living-doc-example-project | [#89 - Test issue 2](89_test_issue_2.md)      | 🔴                | ---            |[GitHub](#) |
-|AbsaOSS           | living-doc-example-project | [#88 - Test issue](88_test_issue.md)          | 🟢                | Todo           |[GitHub](#) |
-|AbsaOSS           | living-doc-example-project | [#41 - Initial commit.](41_initial_commit.md) | 🟢                | Done           |[GitHub](#) |
-|AbsaOSS           | living-doc-example-project | [#33 - Example bugfix](33_example_bugfix.md)  | 🔴                | ---            |[GitHub](#) |
+
+| Organization name| Repository name            | Issue 'Number - Title'         | Linked to project | Project Status | Issue URL  |
+|------------------|----------------------------|--------------------------------|-------------------|----------------|------------|
+| AbsaOSS          | living-doc-example-project | [#89 - Test issue 2](89_test_issue_2.md)      | 🔴 | ---            |[GitHub](#) |
+| AbsaOSS          | living-doc-example-project | [#88 - Test issue](88_test_issue.md)          | 🟢 | Todo           |[GitHub](#) |
+| AbsaOSS          | living-doc-example-project | [#41 - Initial commit.](41_initial_commit.md) | 🟢 | Done           |[GitHub](#) |
+| AbsaOSS          | living-doc-example-project | [#33 - Example bugfix](33_example_bugfix.md)  | 🔴 | ---            |[GitHub](#) |
 ```
 
+**Project Status** can have various values depending on the project, such as: Todo, Done, Closed, In Progress, In Review, Blocked, etc. 
+These values can vary from project to project.
+
+The `---` symbol is used to indicate that an issue has no required project data.
```

Comment:
```suggestion
- **Project Status** can have various values depending on the project, such as: Todo, Done, Closed, In Progress, In Review, Blocked, etc. 
These values can vary from project to project.
- The `---` symbol is used to indicate that an issue has no required project data.
```

Reply:
Suggestion commited in: [458c10b](https://github.com/AbsaOSS/living-doc-generator/pull/27/commits/458c10bb029f1a04947b849a0b7c361827159b0a)